src/tools/perf/perftest.h

```diff
@@ -0,0 +1,113 @@
+/**
+* Copyright (C) NVIDIA 2021.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifndef UCX_PERFTEST_H
+#define UCX_PERFTEST_H
+
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
+
+#include "api/libperf.h"
+#include "lib/libperf_int.h"
+
+#include <ucs/sys/string.h>
+#include <ucs/sys/sys.h>
+#include <ucs/sys/sock.h>
+#include <ucs/debug/log.h>
+
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <netdb.h>
+#include <getopt.h>
+#include <string.h>
+#include <sys/types.h>
+#include <sys/poll.h>
+#include <locale.h>
+
+#if defined (HAVE_MPI)
+#  include <mpi.h>
+
+#elif defined (HAVE_RTE)
```

Comment:
pls add only the needed includes in this H file (e.g which are needed for stuff defined in H file itself). the rest should go to the relevant C files